docs/streams/developer-guide/security.html

```diff
@@ -0,0 +1,186 @@
+        <div class="contents local topic" id="table-of-contents">
+            <p class="topic-title first">Table of Contents</p>
+            <ul class="simple">
+                <li><a class="reference internal" href="#required-acl-setting-for-secure-kafka-clusters" id="id1">Required ACL setting for secure Kafka clusters</a></li>
+                <li><a class="reference internal" href="#security-example" id="id2">Security example</a></li>
+            </ul>
+        </div>
+        <p>Kafka Streams natively integrates with the <a class="reference internal" href="../../kafka/security.html#kafka-security"><span class="std std-ref">Kafka&#8217;s security features</span></a> and supports all of the
+            client-side security features in Kafka.  Streams leverages the <a class="reference internal" href="../../clients/index.html#kafka-clients"><span class="std std-ref">Java Producer and Consumer API</span></a>.</p>
+        <p>To secure your Stream processing applications, configure the security settings in the corresponding Kafka producer
+            and consumer clients, and then specify the corresponding configuration settings in your Kafka Streams application.</p>
+        <p>Kafka supports cluster encryption and authentication, including a mix of authenticated and unauthenticated,
+            and encrypted and non-encrypted clients. Using security is optional.</p>
+        <p>Here a few relevant client-side security features:</p>
+        <dl class="docutils">
+            <dt>Encrypt data-in-transit between your applications and Kafka brokers</dt>
+            <dd>You can enable the encryption of the client-server communication between your applications and the Kafka brokers.
+                For example, you can configure your applications to always use encryption when reading and writing data to and from
+                Kafka. This is critical when reading and writing data across security domains such as internal network, public
+                internet, and partner networks.</dd>
+            <dt>Client authentication</dt>
+            <dd>You can enable client authentication for connections from your application to Kafka brokers. For example, you can
+                define that only specific applications are allowed to connect to your Kafka cluster.</dd>
+            <dt>Client authorization</dt>
+            <dd>You can enable client authorization of read and write operations by your applications. For example, you can define
+                that only specific applications are allowed to read from a Kafka topic.  You can also restrict write access to Kafka
+                topics to prevent data pollution or fraudulent activities.</dd>
+        </dl>
+        <p>For more information about the security features in Apache Kafka, see <a class="reference internal" href="../../kafka/security.html#kafka-security"><span class="std std-ref">Kafka Security</span></a> and the
```

Comment:
Ditto here, need to remove vendor blog posts content, we can consider replacing with a ordinary AK use case.